config/default/kustomization.yaml

```
@@ -8,15 +8,14 @@ namespace: kueue-system
 # field above.
 namePrefix: kueue-
 
-# Labels to add to all resources and selectors.
-commonLabels:
-  control-plane: controller-manager
-
 labels:
-  - pairs:    
+  - pairs:
       app.kubernetes.io/name: kueue
       app.kubernetes.io/component: controller
     includeTemplates: true
+  - pairs:
+      control-plane: controller-manager
+    includeSelectors: true
```

```suggestion
    includeSelectors: true
    includeTemplates: true    
```

Could you check if we need to add this option?

---

No, If I add includeTemplates I can't see changes.

---

And also without includeTemplates the same result.